data/abhijeet-26.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Abhijeet Basfore",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/abhijeet-26.png",
+  "tags": ["javascript", "Springboot", "DataStructures and Algorithms", "react.js", "Microservices","junit/Mockito","Open source"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/AbhijeetBasfore" },
+    { "icon": "FaGithub", "url": "https://github.com/abhijeet-26" },
+    { "icon": "SiHashnode", "url":"https://hashnode.com/@abhijeet26"},
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/abhijeet-26",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AbhijeetBasfore",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/abhijeet-basfore-8526b71b5/",
```

Comment:
There's something wrong with the LinkedIn URL, it takes me to 404 page

Reply:
please check now